source/client-side-encryption/tests/README.rst

```diff
@@ -1039,15 +1054,15 @@ Invalid KMS Certificate
       {
          "region": "us-east-1",
          "key": "arn:aws:kms:us-east-1:579766882180:key/89fcc2c4-08b0-4bd9-9f25-e30687b580d0",
-         "endpoint": "mongodb://127.0.0.1:8000",
+         "endpoint": "127.0.0.1:8000",
       }
 
-   Expect this to fail with an exception with a message containing the string: "expired".
+   Expect this to fail with an exception with a message referencing an expired certificate. This message will be language dependent.
```

Comment:
I could add an example of Go or Python's error message here. Let me know if that would be helpful.

Reply:
Yes I think an example would be helpful. For the expired test Python's error would be `"certificate verify failed: certificate has expired (_ssl.c:1129)"`. For the wrong-host test the error is `"certificate verify failed: IP address mismatch, certificate is not valid for '127.0.0.1'. (_ssl.c:1129)""`

Reply:
Thanks. Added examples from Python and Go.